src/Box.js

```diff
@@ -8,6 +8,7 @@ const Box = styled.div`
   ${width}
   ${color}
   ${align}
+  ${borderRadius}
```

Comment:
Don't think you're actually using this are you?

Reply:
Nope